recipe/meta.yaml

```diff
@@ -1,18 +1,27 @@
-{% set version = "8.1.0.77" %}
+{% set version = "8.2.0.53" %}
+
+# cuDNN v8.2.0 supports CUDA versions 10.2 - 11.3
+# For CUDA 10.2 there is a separate cuDNN distribution and a link for download
+# For CUDA 11.x there is one cuDNN distribution that works with all CUDA 11 versions
+{% if cuda_compiler_version[:2] == "11" %}
```

Comment:
Looks like accessing this Jinja variable would cause `cudnn` leaking into the ci support filename...🤔